mlir/utils/jenkins/static-checks/premerge-checks.py

```diff
@@ -17,21 +17,21 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 import os
+import sys
 import re
 import subprocess
 from typing import Tuple, Optional
 import pathspec
 import unidiff
+import argparse
 
 
 def get_diff(base_commit) -> Tuple[bool, str]:
-  r = subprocess.run(f'/opt/rocm/llvm/bin/git-clang-format {base_commit}', shell=True)
-  if r.returncode != 0:
-    r = subprocess.run(f'git checkout -- .', stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
+  diff_run = subprocess.run(f'/opt/rocm/llvm/bin/git-clang-format --diff {base_commit}', shell=True)
+  if diff_run.returncode != 0:
     return False, ''
-  diff_run = subprocess.run(f'git diff -U0 --no-prefix --exit-code', stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
```

Comment:
I think the point of this diff call is to show the changes that come up when you run clang-format. We might want to keep it, but it's not urgent given our workflow.

Reply:
Thats spelled git clang-format --diff and its still maintainined in the same way and one does not need to do the changes and revert

Reply:
(See my new diff_run var)

Reply:
Ah, cool. in which case, we're good